nikola/data/themes/base-jinja/templates/post_header.tmpl

```diff
@@ -44,7 +44,7 @@
             {% endif %}
             {{ html_sourcelink() }}
             {% if post.meta('link') %}
-                    <p class="linkline"><a href='{{ post.meta('link') }}'>{{ messages("Original site") }}</a></p>
+                    <p class="linkline"><a href="{{ post.meta('link') }}">{{ messages("Original site") }}</a></p>
```

Comment:
Please don't edit the -jinja themes directly, they are autogenerated from the Mako versions using the jinjify script